.github/workflows/ci.yml

```diff
@@ -0,0 +1,167 @@
+name: "CI"
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
So this action looks like it tries to create a new release w/ the version from package.json - but we're running it on every push to main?  Seems like it's going to cause clashes pretty easily.